frontend/src/components/SummaryDeclaration.vue

```diff
@@ -107,88 +107,121 @@
                 />
 
                 <div v-else>
-                  <v-expansion-panel v-if="facility?.facilityInfo" variant="accordion" value="facility-information">
-                    <FacilityInformationSummary
-                      :facility-info="facility?.facilityInfo"
-                      :funding="facility?.funding"
-                      :facility-id="facility.facilityId"
-                      :ccfri-status="facility?.ccfri?.ccfriOptInStatus"
-                      :ecewe-status="facility?.ecewe?.optInOrOut"
-                      :license-categories="facility?.licenseCategories"
-                      :provider-type="summaryModel?.application?.organizationProviderType"
-                      :change-rec-guid="facility?.changeRequestId"
-                      :program-year-id="summaryModel?.application?.programYearId"
-                      @is-summary-valid="isFormComplete"
-                    />
-                  </v-expansion-panel>
-                  <v-expansion-panel variant="accordion" value="ccof-summary">
-                    <div v-if="!facility.funding || isRenewal" />
-                    <div v-else>
-                      <CCOFSummaryFamily
-                        v-if="
-                          summaryModel?.application?.organizationProviderType === ORGANIZATION_PROVIDER_TYPES.FAMILY
-                        "
-                        :funding="facility.funding"
+                  <v-expansion-panels>
```

Comment:
Hi Mael, could you also update the expandAllPanels() function to align with your new changes? The expected behavior is that if you add /printable to the end of the summary declaration URL, all panels should automatically expand.